content/4_Gold/Divisibility.mdx

```diff
@@ -507,6 +507,167 @@ $$
 \gcd(a_1, a_2, a_3, a_4) = \gcd(a_1, \gcd(a_2, \gcd(a_3, a_4))).
 $$
 
+## Euler's Totient Function
+
+<FocusProblem problem="gcdex" />
```

Comment:
maybe move this down to above the solution?

Reply:
Go ahead, do what you want

Reply:
Ok I moved the problem down